src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -1,17 +1,21 @@
 package com.gu.fastly
 
-import org.apache.commons.codec.digest.DigestUtils
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder
+import com.amazonaws.services.cloudwatch.model.{MetricDatum, PutMetricDataRequest, StandardUnit}
 import com.amazonaws.services.kinesis.clientlibrary.types.UserRecord
 import com.amazonaws.services.kinesis.model.Record
 import com.amazonaws.services.lambda.runtime.events.KinesisEvent
-import okhttp3._
 import com.gu.crier.model.event.v1._
+import okhttp3._
+import org.apache.commons.codec.digest.DigestUtils
+
 import scala.collection.JavaConverters._
 
 class Lambda {
 
   private val config = Config.load()
   private val httpClient = new OkHttpClient()
+  private val cloudWatchClient = AmazonCloudWatchClientBuilder.defaultClient
```

Comment:
This appears to be no worse that what the S3 client is doing.
Does the AWS client library have any awareness of the auth and region context the Lambda is running in?